projects/core/src/user/facade/user.service.ts

```diff
@@ -290,6 +246,13 @@ export class UserService {
     );
   }
 
+  /**
+   * Returns addresses state
+   */
+  getAddressesState(): Observable<fromStore.UserAddressesState> {
```

Comment:
We shouldn't expose the whole state in facades. Instead, we should expose all the necessary parts as separate methods.

Reply:
Method removed.